source/write/update.txt

```diff
@@ -224,6 +224,6 @@ API Documentation
 To learn more about any of the methods or types discussed in this
 guide, see the following API documentation:
 
-- `MongoDB\\Collection::updateOne() <{+api+}/method/MongoDBCollection-updateOne>`__
-- `MongoDB\\Collection::updateMany() <{+api+}/method/MongoDBCollection-updateMany>`__
-- `MongoDB\\UpdateResult <{+api+}/class/MongoDBUpdateResult>`__
\ No newline at end of file
+- :phpmethod:`MongoDB\Collection::updateOne()`
+- :phpmethod:`MongoDB\Collection::updateMany()`
+- :phpclass:`MongoDB\UpdateResult`
```

Comment:
Q: are there also places that need to use the :php: role to reference extension API docs?

Reply:
fixed the remaining ones!